Vagrantfile

```diff
@@ -5,35 +5,22 @@ VAGRANTFILE_API_VERSION = '2'
 
 @script = <<SCRIPT
 DOCUMENT_ROOT_ZEND="/var/www/zf/public"
```

Comment:
Append this if you want for change the default workdir. this allow to remove all those `cd /var/www ;` from vagrant ssh commands

```
    if ! grep "cd /vagrant" /home/vagrant/.profile > /dev/null; then
      echo "cd /vagrant" >> /home/vagrant/.profile
    fi
```